textBasedAdventure.js

```diff
@@ -1,5 +1,149 @@
 const readline = require('readline-sync')
 
-let nameInput = readline.question("Enter your name: ")
+var readlineSync = require('readline-sync');
+var userName = readlineSync.question('May I have your name? ');
 
-console.log(`Hello ${nameInput}!  Welcome to my game.`)
+console.log('Hi ' + userName + '!');
+
+
+function gamePlay () {
+  let userInput;
+
+  while (userInput !== 'yes'){
+      userInput = readline.question(`${userName} would you like to find out which type of runner you are? 'yes' or 'no'`)
+  } 
+  console.log(`I am so glad you said yes`);
+
+  let worldPick;
+  worldPick = readline.question(`${userName} on a scale from 1-3, how much do you run? (note: 1 being the least amount, 3 the highest)`);{
+console.log(worldPick)}
+    if(worldPick === '2'){
+      escapedConvict();
+    }else if(worldPick === '1'){
+      portableDJ();
+    }else if(worldPick === '3'){
+      enduranceAudioBookRunner();
+    }else(console.log(readline.question(`${userName} on a scale from 1-3, how much do you run?`)))
+  }
+
+function enduranceAudioBookRunner() {
```

Comment:
For future projects, write function using ES6 syntax:
`const enduranceAudioBookRunner = () => {}`